docs/configuration.md

```diff
@@ -55,6 +55,10 @@ Check out the [example configuration files](https://github.com/shinesolutions/ae
 | aws.tags | An array of `Key` and `Value` pairs for tagging AWS resources (e.g. EC2 instance, AMI, EBS volume) created by Packer AEM following your organisation's tagging standard | Optional | None |
 | aws.certificate_arn | The ARN of the Certificate in the [AWS Certificate Manager (ACM)](https://console.aws.amazon.com/acm/home) | Mandatory | |
 | aws.certificate_key_arn | The ARN of the secret containing TLS certificate's secret key in the [AWS Secrets Manager](https://console.aws.amazon.com/secretsmanager/home) | Optional | |
-| aws.secrets_manager_enabled | A boolean value to indicate if AEM OpenCloud can provision AWS Secrets Manager resource, which is currently supported for storing TLS certificate's private key  | Optional | true |
+| aws.resources.stack_name | Appended string for Packer AEM Resources stack-name created by `make create-aws-resources`| Optional | `packer-aem-resources-stack`|
+| aws.resources.s3_bucket | Name of bucket created by `make create-aws-resources` | Mandatory | `overwrite-me` |
+| aws.enable_secrets_manager | A `'true'` or `'false'` string value to indicate if AEM OpenCloud can provision AWS Secrets Manager resource, which is currently supported for storing TLS certificate's private key  | Optional | `'true'` |
```

Comment:
Could these 3 new params that are aws-resources specific be placed under `aws.resources` prefix? That way it'll be more obvious for users that the configs are related to resources.

Reply:
Done!